samples/debug/ppi_trace/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _ppi_trace_sample:
+
+PPI trace
+#########
+
+The PPI trace sample shows how to trace hardware events on GPIO pins.
+It uses the :ref:`ppi_trace` module.
+
+Overview
+********
+
+The sample initializes trace pins to observe the following hardware events:
+
+* RTC Compare event (``NRF_RTC_EVENT_COMPARE_1``)
+* RTC Tick event (``NRF_RTC_EVENT_TICK``)
+* Low frequency clock (LFCLK) Started event (``NRF_CLOCK_EVENT_LFCLKSTARTED``)
+* Radio activity during *Bluetooth* advertising (available only for Bluetooth capable devices)
+
+The sample sets up a :ref:`zephyr:counter_interface` to generate an event every 50 ms.
```

Comment:
What event does it generate? RTC Tick?

Reply:
no, it generates `NRF_RTC_EVENT_COMPARE_1`

Reply:
added